_posts/2019-11-12-experience-agile-2019.md

```diff
@@ -0,0 +1,307 @@
+> The Toyota Flow System is the first to externalize it with tools and in a contextual setting. 
+> The next step will be testing it in the field in different contextual settings, to see what works and what doesn't. 
+> It will be exciting to see how it evolves, which we are sure it will. 
+> That’s the beauty of it.
+
+## The Agile Futurist, by Mario Moreira
+This was an energizing talk, giving us a view on the trends in the Agile movement for the coming ten years. 
+Mario Moreira is an enthusing and influential Agile transformation Leader, Agile enterprise coach and change agent who has written four books a.o. *Being Agile: Your Roadmap to Successful Adoption of Agile*, and *The Agile enterprise: Building and Running Agile organizations*.
+
+What is Agile?... is what he asked the eager crowd. 
+- It's a set of Values and Principles 
+- focused on the delivery of value to the customer
+
+What is Agile not?... on the other hand.
+- a certification
+- a tool
+- a silver bullet
+- a process
+- it's also not merely *redefining roles*
+- and... it is definitely not *undisciplined*
+
+### Agile trends for 2029
+> Agile’s Future?
+> - We will Bring Back the Basics
+> - We will Stop doing Agile for Agile’s sake
+> - We will Lead with Uncertainty
+> - We will Apply Agile across an Enterprise
+> - We will Solve Holistic Problems
+
+#### Bring Back the Basics.
+
+We will all go back to the core, to rediscover and be inspired by the central purpose and initial intent of the Agile movement.
+Returning to the core implies that we need to make sure we understand the heart of the [Agile manifesto](https://agilemanifesto.org/){:target="_blank" rel="noopener noreferrer"}, that we try and urge to live the principles and lead with the items on the left in mind. 
+
+{:start="2"}
+2. Stop doing Agile for Agile’s sake.
+
+Agile is not the goal or outcome. 
+So let's stop talking about it as if it were.
+Rather we should look at the purpose for applying Agile.
+- Better interactions
+- Better working software (or services and products for that matter)
+- Better customer collaboration
+- Ability to respond to change
+
+Instead, people will focus on better Business Outcomes:
+- Deliver increased Customer **Value**
+- Optimize the **Flow** for faster delivery
+- Increase **Quality** with Feedback Loops
+
+{:start="3"}
+3. Leading for uncertainty 
+
+Because leading for uncertainty is the smarter thing to do, in a world where we as customers don't always know what we want, we as product creators don't always know exactly how to build the product, and things and people tend to change along the way.
+
+Uncertainty requires the right behavior and mindset to tackle it: e.g. cultivate and kindle a discovery mindset, experimental and incremental thinking, and implementing feedback loops.
+We all need to practice and keep on practicing, to walk the talk, and eventually lead by example. 
+
+{:start="4"}
+4. Agile throughout the enterprise
```

Comment:
Was there supposed to be a section following under this fourth element like with the previous ones?